datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx

```diff
@@ -120,68 +171,177 @@ export const EntityHeader = () => {
+                            <Link to={entityRegistry.getEntityUrl(EntityType.Container, container?.urn)}>
+                                <PlatformDivider />
+                                <ContainerIcon
+                                    style={{
+                                        color: ANTD_GRAY[9],
+                                    }}
+                                />
+                                <ContainerText>
+                                    {entityRegistry.getDisplayName(EntityType.Container, container)}
+                                </ContainerText>
+                            </Link>
+                        )}
+                        {entityCount && entityCount > 0 ? (
+                            <>
+                                <PlatformDivider />
+                                <EntityCountText>{entityCount.toLocaleString()} entities</EntityCountText>
+                            </>
+                        ) : null}
+                    </PlatformContent>
+                    <div style={{ display: 'flex', justifyContent: 'left', alignItems: 'center' }}>
+                        <Link to={entityPath}>
+                            <EntityTitle level={3}>{entityData?.name || ' '}</EntityTitle>
                         </Link>
-                    )}
-                    {entityCount && entityCount > 0 ? (
-                        <>
-                            <PlatformDivider />
-                            <EntityCountText>{entityCount.toLocaleString()} entities</EntityCountText>
-                        </>
-                    ) : null}
-                </PlatformContent>
-                <div style={{ display: 'flex', justifyContent: 'left', alignItems: 'center' }}>
-                    <Link to={entityPath}>
-                        <EntityTitle level={3}>{entityData?.name || ' '}</EntityTitle>
-                    </Link>
-                    {entityData?.health && (
-                        <EntityHealthStatus
-                            status={entityData?.health.status}
-                            message={entityData?.health?.message || undefined}
-                        />
-                    )}
-                </div>
-            </MainHeaderContent>
-            {hasExternalUrl && (
-                <ExternalLinkButton href={externalUrl} onClick={sendAnalytics}>
-                    View in {platformName}
-                </ExternalLinkButton>
-            )}
-            <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
-                <Button
-                    icon={copiedUrn ? <CheckOutlined /> : <CopyOutlined />}
-                    onClick={() => {
-                        navigator.clipboard.writeText(urn);
-                        setCopiedUrn(true);
-                    }}
-                />
-            </Tooltip>
-        </HeaderContainer>
+                        {entityData?.deprecation?.deprecated &&
```

Comment:
lets do this. above create a new variable

const hasDetails = (entityData?.deprecation?.note !== '' ||
                                entityData?.deprecation?.decommissionTime !== null)
 
then inside the content box we can do this:

!hasDetails && <current line 276-295> || 'No additional details' 

then you can get rid of the long conditional on 269, as well as the line 303-310


entityData?.deprecation?.deprecated

Reply:
Fixed and committed my changes